### ISSUE TITLE
The published package "php-swoole" 6.0.0 seems to be in a pre-release/alpha stage

### ISSUE THREAD

**Issue by pixelart7** [oerdnj/deb.sury.org#2152]
**Frequently asked questions**
- [x] I have read [Frequently Asked Questions](https://github.com/oerdnj/deb.sury.org/wiki/Frequently-Asked-Questions)
- [x] I have looked at the list of the existing issues (including closed issues) and searched if my issue has been already reported
- [x] I have tried to resolve the issue myself and will describe what I did in clear and concise manner

**Describe the bug**
The php-swoole 6.0.0 package published on 2024-06-24 appears to be from a pre-release version of Swoole. It is stated as a test version / alpha and not recommended for production use (See on [PECL](https://pecl.php.net/package/swoole) / See on [swoole/swoole-src's 6.0.0-alpha release note](https://github.com/swoole/swoole-src/releases/tag/v6.0.0-alpha)).

**To Reproduce**
The investigation into the published version being a pre-release should be straightforward.

**Your understanding of what is happening**
I understand that `apt install` will install extensions that you've built and distributed here. However, with Swoole 6, there might be some breaking changes in how PostgreSQL connects with SSL. Since Swoole 6 is still in alpha, this breaking change might have been overlooked or intended, which is where I'm uncertain.

**What steps did you take to resolve issue yourself before reporting it here**
1. Investigated our existing deployment (that uses a base Docker image which uses this repo for PHP extensions) and our newest deployment on Tuesday, the day after Swoole 6 went live in the repo.
2. Found the difference in version.
3. Discovered a [StackOverflow thread](https://stackoverflow.com/questions/78669556/possible-bug-with-php-pdo-and-with-postgresql) that points to how Swoole 6 might break PostgreSQL connections (see "Additional context" section).

**Expected behavior**
The repository should provide a stable version of Swoole.

**Distribution (please complete the following information):**
 - OS: Ubuntu
 - Architecture: amd64
 - Repository: Ubuntu PPA

**Package(s) (please complete the following information):**
```
php8.2-swoole:
  Installed: 6.0.0-2+ubuntu22.04.1+deb.sury.org+1
  Candidate: 6.0.0-2+ubuntu22.04.1+deb.sury.org+1
  Version table:
 *** 6.0.0-2+ubuntu22.04.1+deb.sury.org+1 500
        500 http://ppa.launchpad.net/ondrej/php/ubuntu jammy/main amd64 Packages
        100 /var/lib/dpkg/status
```

**Additional context**
To reproduce the issue I encountered:
1. Install both pgsql (php8.2-pgsql 8.2.20-2+ubuntu22.04.1+deb.sury.org+1) and swoole (php8.2-swoole 6.0.0-2+ubuntu22.04.1+deb.sury.org+1)
2. Try connecting to a PostgreSQL server via SSL
3. The following error will be reported: "SQLSTATE[08006] [7] could not send SSL negotiation packet: Resource temporarily unavailable"

Initially, this error seemed unrelated. However, in another deployment using the same Dockerfile, I checked and found that the only difference was the Swoole version. The working version was 5.1.2.

**Comment by webard** [comment]
For now I pinned version 5.1.* by creating file /etc/apt/preferences.d/swoole with content

```
Package: php8.3-swoole
Pin: version 5.1.*
Pin-Priority: 999
```

but is seems that in Sury repo is available version 5.1.2, and the latest is 5.1.3.

**Comment by oerdnj** [comment]
Why on earth isn't the version named 6.0.0alpha1 or something...

Downgraded to 6.0.0+really+5.1.3

**Comment by yoyoyeahyoyo** [comment]
I am still getting the same bug after installing ( 6.0.0+really+5.1.3-1).

**Comment by wjdix** [comment]
I believe that the swoole regression was introduced in 5.1.3; based on the swoole repository https://github.com/swoole/swoole-src/issues/5313, it seems like a 5.1.4 release will need to happen to fix this issue. Is there a possibility to make swoole 5.1.2 available again until 5.1.4 is released?

**Comment by jassingh9** [comment]
5.1.4 is now released: #https://github.com/swoole/swoole-src/releases/tag/v5.1.4 can this be pushed to the repo?
